app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -821,6 +821,54 @@
 					}
 				}
 			]
+		},
+		"viz_tMiiAsvK": {
+			"type": "viz.text",
+			"options": {
+				"content": "Malware Detection events received",
+				"fontSize": 12,
+				"fontWeight": "bold"
+			}
+		},
+		"viz_JSPTHVku": {
+			"type": "viz.text",
+			"options": {
+				"content": "Last 24 hours",
+				"fontSize": 10
+			}
+		},
+		"viz_89oyYK2v": {
+			"type": "splunk.singlevalue",
+			"dataSources": {
+				"primary": "ds_1AuG6Zgj_ds_IjDQvLZj_ds_q5kL8bG5_ds_lDtnkMeN"
+			},
+			"options": {
+				"unit": "",
+				"trendColor": "> trendValue | rangeValue(trendColorEditorConfig)",
+				"sparklineDisplay": "off",
+				"trendFontSize": 14
+			},
+			"title": "",
+			"context": {
+				"trendColorEditorConfig": [
+					{
+						"value": "#118832",
+						"to": 1
+					},
+					{
+						"value": "#d41f1f",
+						"from": 1
+					}
+				]
+			},
+			"eventHandlers": [
+				{
+					"type": "drilldown.customUrl",
+					"options": {
+						"url": "/app/redhat_insights/events?form.timepicker.earliest=-24h%40h&form.timepicker.latest=now&form.field_account=*&form.field_application=Malware-detection&form.filter_system=*"
```

Comment:
Aren't we changing the application to `Malware`? Shouldn't the URL reflect that?

Reply:
You're right, forgot to change here... fixed.